docs/the-basics.rst

```diff
@@ -1050,6 +1056,103 @@ Annotations (:ref:`Annot`) on pages can be retrieved with the `page.annots()` me
     - :meth:`Page.annots`
 
 
+--------------------------
+
+
+
+.. _The_Basics_Redacting:
+
+Redacting content from a **PDF**
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Redactions are special types of annotations which can be marked onto a document page to denote an area on the page which should be securely removed. After marking an area with a rectangle then this area will be marked for *redaction*, once the redaction is *applied* then the content is securly removed.
+
+For example if we wanted to redact all instances of the name "Jane Doe" from a document we could do the following:
+
+.. raw:: html
+
+    <pre>
+        <code class="language-python" data-prismjs-copy="Copy">
+            import fitz
+
+            # Open the PDF document
```

Comment:
How come code in this file `docs/the-basics.rst` uses `raw:: html <pre><code>...` here? Code samples elsewhere use normal `rst`'s code syntax?

Reply:
When "The Basics" page was first created our code snippets elsewhere didn't have the "Copy to clipboard" functionality and it was important to have that on "The Basics" section for people to easily grab. However, since: commit: 50a33001e1edfbd53e414310c6b6d3eb075d090f in December this is no longer an issue as we have a proper Sphinx extension to do the work!

So, yes, good spot I _think_ we can happily get rid of all the prism JS stuff and revisit this.